compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -215,61 +215,72 @@ OMR::ARM64::CodeGenerator::doRegisterAssignment(TR_RegisterKinds kindsToAssign)
       }
    }
 
+void
+OMR::ARM64::CodeGenerator::generateBinaryEncodingPrePrologue(TR_ARM64BinaryEncodingData &data)
+{
```

Comment:
```suggestion
   {
```

Reply:
Corrected indentation. Sorry about that.